src/utils/channelUtils.js

```diff
@@ -0,0 +1,32 @@
+const generateUniqueChannelName = (guild, channelPrefix) => {
+    const maxChannel = guild.channels.cache.filter(channel => channel.name.startsWith(channelPrefix + '-'))
+        .map(channel => parseInt(channel.name.split('-')[1], 10)).sort().pop() || 0;
+    return `${channelPrefix}-${maxChannel + 1}`;           
```

Comment:
Maybe we can change this to use a randomly generated name instead of iterating through existing channels with prefix to get maxChannel.